go.mod

```diff
@@ -470,7 +472,6 @@ replace (
 	github.com/sigstore/cosign/v2 => github.com/stackrox/cosign/v2 v2.0.0-20231206143943-626efb842da7
 
 	github.com/tecbot/gorocksdb => github.com/DataDog/gorocksdb v0.0.0-20200107201226-9722c3a2e063
-	go.uber.org/zap => github.com/stackrox/zap v1.15.1-0.20230918235618-2bd149903d0e
```

Comment:
do we no longer require the forked changes? were they merged upstream? just curious about why we can finally remove this

Reply:
I created a PR to investigate it https://issues.redhat.com/browse/ROX-23111
There are 2 features that we use (ordering of fields and not verbose errors output) 
I removed it here to see if it builds

Reply:
```
go mod why -m go.uber.org/zap
# go.uber.org/zap
github.com/stackrox/rox/operator
go.uber.org/zap
```

Reply:
does this have to be done in this PR or can this be put into a separate one?

Reply:
I have a separated PR for that 
- https://github.com/stackrox/stackrox/pull/10375